CREDITS

```diff
@@ -7,9 +7,11 @@ Colin Hanrahan
 Willem Hueffed
 Nate Kremer-Herman
 Maria Milkowski
+Anna Muller
 Joshua Palicka
 Mariana Shuman
 Lucas Vorkoper
+random
```

Comment:
I don't think "random" is supposed to be here.

Reply:
That was a test that I forgot to delete.